### DISCUSSION TITLE
ipset states need to persist ipsets across reboots.

### DISCUSSION THREAD

**Discussion by roskens** [saltstack/salt#67649] [Feature Requests]
### Description of Issue/Question

The iptables states have a save option to optionally run `iptables save > /etc/sysconfig/iptables` when changes are made. The ipset states should also behave similarly when `save: true` is added to the state.

Alternatively, the ipset state documentation could mention that ipset's are not persisted on reboot and you need to modify /etc/sysconfig/ipset-config and set IPSET_SAVE_ON_STOP="yes"

### Setup
ipset.sls
```
SALT-MASTER:
  ipset.set_present:
    - set_type: hash:ip
```

### Steps to Reproduce Issue

$ salt $MINIONID state.apply ipset
$ salt $MINIONID system.reboot at_time=1
$ salt $MINIONID cmd.run "ipset list"

### Versions Report
Salt Version:
           Salt: 2018.3.3

Dependency Versions:
           cffi: 1.6.0
       cherrypy: Not Installed
       dateutil: 1.5
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: 0.26.3
        libnacl: 1.6.1
       M2Crypto: 0.28.2
           Mako: 0.8.1
   msgpack-pure: Not Installed
 msgpack-python: 0.5.1
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.26.4
         Python: 2.7.5 (default, May 31 2018, 09:41:32)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: redhat 7.5 Maipo
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-862.6.3.el7.x86_64
         system: Linux
        version: Red Hat Enterprise Linux Server 7.5 Maipo

**Comment by dwoz** [comment]
@roskens thanks for reporting this!
